legend-engine-xt-relationalStore-pure/src/main/resources/core_relational/relational/sqlQueryToString/testSuite/dynaFunctions/string.pure

```diff
@@ -282,7 +282,7 @@ function <<paramTest.Test>> meta::relational::tests::dbSpecificTests::sqlQueryTe
 
 function <<paramTest.Test>> meta::relational::tests::dbSpecificTests::sqlQueryTests::dynaFunctions::matches::testMatchesNonAlphaNumeric(config:DbTestConfig[1]):Boolean[1]
 {
-  let dynaFunc = ^DynaFunction(name='matches', parameters=[^Literal(value='Blo$ggs'), ^Literal(value='[A-Za-z0-9]*')]);
+  let dynaFunc = ^DynaFunction(name='matches', parameters=[^Literal(value='Blo$ggs'), ^Literal(value='^[A-Za-z0-9]*$')]);
```

Comment:
Test was wrong. `Blo$ggs` is matching `[A-Za-z0-9]*` as it contains a series of alphanumerical characters (`ggs`)